.github/workflows/reusable-doc-test.yaml

```diff
@@ -53,56 +31,13 @@ jobs:
     env:
       CURR_REPO: ${{ github.event.repository.name }}
     outputs:
-      # The name of the untagged docker images to be created by this workflow
-      IMAGE_NAME: ${{ steps.set_image_name.outputs.IMAGE_NAME }}
-      REPO_NAME: ${{ steps.set_image_name.outputs.REPO_NAME }}
-      # The tag to use for the doclinttest images
-      DOCLINTTEST_TAG: ${{ steps.set_doclinttest_tag.outputs.DOCLINTTEST_TAG }}
-      # The image to use for doclinttest jobs
-      DOCLINTTEST_IMAGE: ${{ steps.set_doclinttest_image.outputs.DOCLINTTEST_IMAGE }}
       # Flags for enabling/disabling various parts of the workflow
       # Default to enabled unless overridden by a repository-level variable
       CI_BUILD_SPHINX_HTML: ${{ steps.get_job_flags.outputs.CI_BUILD_SPHINX_HTML }}
       CI_BUILD_SPHINX_PDF: ${{ steps.get_job_flags.outputs.CI_BUILD_SPHINX_PDF }}
       CI_TEST_INTERFACES: ${{ steps.get_job_flags.outputs.CI_TEST_INTERFACES }}
       CI_TEST_PYTEST_SHORT: ${{ steps.get_job_flags.outputs.CI_TEST_PYTEST_SHORT }}
```

Comment:
Note I set all of these to false specifically in this geoips_ci repo.  I think that is appropriate, and now doc-test.yaml passes (because it skips all the tests...)